script/install.sh

```diff
@@ -474,6 +474,17 @@ resolve_latest() {
             logV "Determining latest ldc-beta version ($url)."
             COMPILER="ldc-$(fetch $url)"
             ;;
+        ldc-nightly)
+            local url=http://thecybershadow.net/d/github-ldc
+            logV "Finding latest ldc-nightly binary package (at $url)."
+            local package
+            package="$(fetch $url)"
+            if [[ $package =~ ^([0-9a-f]*)[[:space:]]([0-9]*)$ ]]; then
+                COMPILER="ldc-${BASH_REMATCH[1]}-${BASH_REMATCH[2]}"
```

Comment:
This is going to always fail for Windows, and also when Linux and macOS end up finalizing the build on different dates. See current binaries at https://github.com/ldc-developers/ldc/releases/tag/CI

The different dates issue can be fixed by LDC's CI though.

Reply:
Can't you just exclude the date from the filename then?
(The install script doesn't support Windows, so that's not a big concern for now).

Reply:
@kinke Any particular reason for adding the date? If it's only to help humans a little bit, then let's drop it if we cannot get macos and linux to sync better.

Reply:
Yeah it's just to aid humans. If I don't clean up the packages for a longer while, there may be hundreds of files, so the date suffix isn't that helpful then anyway.

Reply:
It's a bummer that we cannot automate the package removal process :(